src/noisepy/seis/plotting_modules.py

```diff
@@ -192,6 +192,7 @@ def plot_substack_cc(
 
     # for spair in spairs:
     for src_sta, rec_sta in sta_pairs:
+        ccs = cc_store.read(ts, src_sta, rec_sta)
```

Comment:
The code will actually loop through all the station pairs instead of repeating the first pair.